core/src/test/java/uk/gov/justice/services/core/handler/registry/HandlerRegistryTest.java

```diff
@@ -35,41 +36,27 @@
     private HandlerRegistry registry;
 
     @Test
-    public void shouldHandleActionWithAValidAsynchronousHandler() {
-        createRegistry(new TestCommandHandler());
-        assertThat(registry.canHandleAsynchronous(COMMAND_NAME), equalTo(true));
-        assertThat(registry.canHandleSynchronous(COMMAND_NAME), equalTo(false));
-    }
-
-    @Test
-    public void shouldHandleActionWithAValidSynchronousHandler() {
-        createRegistry(new TestCommandHandlerWithSynchronousHandler());
-        assertThat(registry.canHandleSynchronous(COMMAND_NAME), equalTo(true));
-        assertThat(registry.canHandleAsynchronous(COMMAND_NAME), equalTo(false));
-    }
-
-    @Test
     public void shouldReturnTheRegisteredAsynchronousHandler() {
         createRegistry(new TestCommandHandler());
-        assertThat(registry.getAsynchronous(COMMAND_NAME), notNullValue());
+        assertThat(registry.get(COMMAND_NAME, false), notNullValue());
```

Comment:
would if be worth moving this into a constant of SYNCHRONOUS = TRUE and doing !SYNCHRONOUS for false for clarity of the test?